seeding/dev/01/02_person.json

```diff
@@ -164,6 +164,300 @@
             "vorname": "migration",
             "familienname": "migration",
             "personalnummer": "9999999"
+        },
+        {
+            "id": 21,
+            "username": "fmeiera",
+            "password": "test",
```

Comment:
"test"-passwords will no longer work, when the passport-requirements are activated again. 
Do these users need a password? If they don't, just omit the field and they will get auto-generated passwords